wp-content/mu-plugins/pantheon/pantheon-updates.php

```diff
@@ -15,12 +16,54 @@ function _pantheon_hide_update_nag() {
 	remove_action( 'network_admin_notices', 'update_nag', 3 );
 }
 
-// Compare the current WordPress version to the latest available
+// Get the latest WordPress version
+function _pantheon_get_latest_wordpress_version() {
+	$core_updates = get_core_updates( array('dismissed' => false) );
```

Comment:
`get_core_updates` is checking to see if WordPress itself has an update, correct? I thought that the original purpose of this hook was to ensure that the nag was not displayed until the WordPress update was available on the Pantheon dashboard.

Reply:
For addressing the case where core is up to date but upstream is not (ln. 63).